urnai/base/savable.py

```diff
@@ -218,8 +195,8 @@ def get_pickleable_dict(self):
 
     def restore_pickleable_attributes(self, dict_to_restore):
         for attr in dict_to_restore:
-            if hasattr(self, 'pickle_black_list'):
-                if attr not in self.pickle_black_list and attr != 'pickle_black_list':
+            if hasattr(self, 'attr_block_list'):
```

Comment:
`attr_block_list` is started in `__init__`, so I can't imagine a case where it isn't in `self`.